### PR TITLE
delete scalar class

### LINKED ISSUE #159: Delete `scaler` class
- Delete the header and implementation file since we do not use it anymore

### PR DESCRIPTION
**Pull request by vwong031**

### CODE REVIEW

**Review by Jyro88 — APPROVED** [review]

File: cruise.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include "cruise.h"
```
Comment (shahdivyank) [inline cruise.cpp]: This class is extinct :( I think you may have added it accidentally from an old branch

File: cruise.h [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#ifndef CRUISE_H
```
Comment (shahdivyank) [inline cruise.h]: Extinct :(

File: states.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#ifndef STATES_H
```
Comment (shahdivyank) [inline states.h]: Extinct :( it was renamed to `global_variables` but you can remove this file

File: warnings.cpp [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#include "warnings.h"
```
Comment (shahdivyank) [inline warnings.cpp]: Extinct :(

File: warnings.h [hunk 5]
```diff
@@ -0,0 +1,29 @@
+#ifndef WARNINGS_H
```
Comment (shahdivyank) [inline warnings.h]: Extinct :(

**Review by shahdivyank — CHANGES_REQUESTED** [review]

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**